CallHome/__init__.py

```diff
@@ -46,32 +47,44 @@ def trn_iter(self):
         annotations = MDTMParser().read(op.join(data_dir, 'callhome.train.mdtm'))
         for uri in sorted(annotations.uris):
             annotation = annotations(uri)
+            uri = 'en_' + uri
+            annotation.uri = uri  # update uri
             yield {
                 'database': 'CallHome',
                 'uri': uri,
                 'annotation': annotation,
+                # annotated part as pyannote.core.Timeline instance
+                'annotated': Timeline(uri=uri, segments=[annotation.get_timeline().extent()])
```

Comment:
`pyannote.database.util.get_annotated` does that for you automatically `annotated` is not provided.

That being said, `timeline.extent()` does not contain the silence before the first segment and after the last one.
If the file is annotated in its entirety (is it?), I'd rather use something like `Segment(0, file_duration)`.
This implies reading `file_duration` from a new file containing `{uri} {duration}` lines.

Reply:
I thought the `timeline.extent()` would return the whole duration of a file. Hmm. So, in this case you have to have the `.uem` files provided for each subset of the data? It's a bit annoying to generate  `.uem` files offline for every database, when pyannote already has access to the annotation file, so it should be able to easily figure out its total duration. And I thought, that's what the `timeline.extent()` would do.

Reply:
Since `Annotation` instances only contains speech segments, there is no way to infer the whole duration from it. This is also why the `gaps` method has a `support` option.

Having a `uem` or getting the whole duration of the file is the only way to handle this.

Reply:
Actually, I think we had this discussion already. But I still have one question, from `MDTM` annotation, you do see the time of each speech segment counts from the beginning of the file, like [here](https://github.com/hbredin/pyannote-db-callhome/blob/master/CallHome/data/callhome.test.mdtm):

`4967 2 275.38 8.28000000000003 speaker NA unknown 4967_B`

So, effectively, you know the `total_duration - the_last_piece_of_silence`. Which is a good enough estimation of the total duration. The last silent piece can be ignored for all intents and purposes.